src/personaclient/PersonaClient.php

```diff
@@ -92,7 +92,11 @@ public function validateToken($params = array()){
         }
 
         $this->getStatsD()->startTiming("validateToken.cache.get");
-        $reply = $this->getCacheClient()->get("access_token:".$cacheKey);
+        $cacheClient = $this->getCacheClient();
+        if($cacheClient)
+        {
+            $reply = $cacheClient->get("access_token:".$cacheKey);
+        }
         $this->getStatsD()->endTiming("validateToken.cache.get");
         if($reply == 'OK'){
```

Comment:
this if statement will throw a php notice because $reply might be undefined, not that its declaration has moved inside the new if block you've added. Might be worth declaring $reply and setting it to null before `if($cacheClient)`